src/framework/components/element/text-element.js

```diff
@@ -141,7 +148,7 @@ Object.assign(pc, function () {
 
         _onParentResize: function (width, height) {
             if (this._noResize) return;
-            if (this._font) this._updateText(this._text);
+            if (this._font) this._updateText();
```

Comment:
Usually better to provide arguments even if they are provided in the default as changing function signature can lead to de-optimization in browser.

Reply:
I made this change because literally all the 10 or so other instances of this._updateText() don't specify the parameter!

Reply:
LOL, ok